Playbooks/playbook-Phishing_Investigation_-_Generic_v2.yml

```diff
@@ -1271,10 +1271,10 @@ tasks:
     ignoreworker: false
   "82":
     id: "82"
-    taskid: dc1f7dd1-a94d-4073-8f52-1e39a193a34a
+    taskid: 29937a18-e577-40bd-8a5a-64bb1e4abac6
     type: regular
     task:
-      id: dc1f7dd1-a94d-4073-8f52-1e39a193a34a
+      id: 29937a18-e577-40bd-8a5a-64bb1e4abac6
```

Comment:
Why is the playbook changed? Does this change have any meaning?

Reply:
Since I changed the task to use the script, it changed the task ID. I changed this ID to be on the safe side but not sure if this is really necessary

Reply:
Ok. No big deal